lib/graphs/discovery-graph.js

```diff
@@ -27,6 +27,13 @@ module.exports = {
             taskName: 'Task.Catalog.dmi'
         },
         {
+            label: 'catalog-ls-huangqiang',
+            taskName: 'Task.Catalog.ls',
+            waitOn: {
+                'catalog-dmi': 'finished'
+            }
+        },
+        {
             label: 'catalog-ohai',
```

Comment:
You may also change this task to wait for the catalog ls task. It might not impact the result here, but please take care in the formal development stories in case the executing sequence is essential.